packages/liquidator/src/liquidationStrategy.js

```diff
@@ -0,0 +1,223 @@
+const assert = require("assert");
+// This module is responsible for sizing liquidation positions.
+// If a defensive strategy is enabled, it will check withdraw positions
+// to see if they require an extension rather than full liquidation.
+module.exports = (config, libs, emit = x => x) => {
```

Comment:
Do you anticipate any reason that this main export could modify its param vars? In that case, I'd prefer to see named variables. In general I think named variables are much more scalable

Reply:
Also, could you add some additional comments here to explain exactly what's expected for each of these in accordance with the other components and modules? emit, for example, expects a very specific structure. see [this](https://github.com/UMAprotocol/protocol/blob/04c3c27d969c3cc7edf28317ef4398bb7b7bb407/packages/monitors/src/BalanceMonitor.js#L6) for an example

Reply:
I consider these arguments to be immutable within the closure in general, but I will document it better. I think its just an opinion on formatting here but I prefer to destructure potentially large objects within the function body rather than within the function signature. I dont care that much though and will change it to be consistent with the rest of the code base.

Reply:
@daywiss I was more so referring to naming `libs` and `emit`. I think `config` is perfectly passed in as an object, but documentation for its expected shape should help readability

Reply:
@daywiss why not use destructuring syntax for config so you can specify in-line what params are expected and their defaults?